src/apps/Hosting/HostingDialog.js

```diff
@@ -50,7 +50,11 @@ const CreateHostingDialog = React.createClass({
 
   handleAddSubmit() {
     const params = this.getHostingParams();
+    const { items } = this.state;
 
+    if (!items.length) {
```

Comment:
changed. Could you please tell me what is the reason to use Lodash instead of the previous version? aren't both enough readable? :P is Lodash faster?